src/Model/ResourceModel/Shopgate/Order.php

```diff
@@ -70,4 +70,27 @@ public function getByOrderId($id)
 
         return $connection->fetchRow($select, $bind);
     }
+
+    /**
+     * Filters for all orders that are not already synchronized to Shopgate
+     *
+     * @return Collection
+     */
+    public function getUnsynchronizedOrders()
+    {
+        $this->getSelect()->where('is_sent_to_shopgate=?', '0');
+        return $this;
+    }
+
+    /**
+     * Filters for all orders that are already cancelled
+     *
+     * @return Collection
```

Comment:
since you are returning `$this` I am assuming it should be the current class, which is `Order`?

Reply:
done